app/views/admin/_header.html.erb

```diff
@@ -35,7 +35,11 @@
   <% if policy(AdminBlacklistedEmail).index? %>
     <li><%= link_to ts("Banned Emails", key: "header"), admin_blacklisted_emails_path %></li>
   <% end %>
-  <li><%= link_to ts("Spam", key: "header"), admin_spam_index_path %></li>
+  
+  <% if policy(:spam).index? %>
+     <li><%= link_to ts("Spam", key: "header"), admin_spam_index_path %></li>
+   <% end %>
```

Comment:
I think the indenting acquired some extra spaces on lines 40 and 41

Reply:
fixed